benchmarking/README.md

```diff
@@ -0,0 +1,31 @@
+# Kakarot Benchmarking
+
+The directory contains scripts which we use to benchmark Kakarot, as of now we
+are using aritillery to create virtual users and test the RPC.
+
+NOTE: If you have a `.env` file at the root of this repository, then it is suggested to rename it to something else when you run these benchmarks, the reason being that the RPC will always override variables from `.env`, which can lead to benchmarks not working.
+
+Steps to benchmark:
+
+- install all submodules in the project root:
+  - git submodule update --init --recursive
+- install Kakarot dependencies and compile:
+  - from the project root:
+    - cd lib/kakarot
+    - poetry install { creating a venv is advised }
+    - STARKNET_NETWORK=madara poetry run python ./scripts/compile_kakarot.py
```

Comment:
we'd need to deploy as well I guess. But I'm wondering if we could just use `docker compose up` for all the setting

Reply:
The reason for not using docker was to get as close to real TPS as possible. I know docker doesn't emulate, but I just wanted to avoid any overhead that could be caused by it.

We can later on experiment with a setup via docker and if we see similar performance then we can shift!

Lemme know what you think about it.